src/app/assets/languages/en.json

```diff
@@ -220,11 +220,11 @@
           "max-count-tooltip": "Configure the total number of entries that the cache can contain. When the cache reaches this threshold, {{brandname}} removes entries according to the eviction strategy that you configure.",
           "max-count-helper-invalid": "You must specify a positive number",
           "eviction-strategy": "Eviction strategy",
-          "eviction-strategy-tooltip": "Specify how {{brandname}} removes entries when the cache reaches the threshold that you configure. The REMOVE strategy evicts older entries to make space for new ones. The EXCEPTION strategy configures {{brandname}} to log an exception and prevent new entries from being created.",
+          "eviction-strategy-tooltip": "Specify how {{brandname}} should remove entries when the cache reaches the threshold that you configure. The REMOVE strategy evicts older entries to make space for new ones. The EXCEPTION strategy configures {{brandname}} to log an exception and prevent new entries from being created.",
```

Comment:
Why "should"?

Reply:
My interpretation was that eviction "strategy" was a choice that the user was making, and that "should" reflected that more. But this doesn't necessarily add new value if you prefer it without.